docs/en/ingest-management/fleet/add-fleet-server-on-prem.asciidoc

```diff
@@ -0,0 +1,154 @@
+* Add hosts including <<add-fleet-server-on-prem-hosts,{fleet-server} hosts and {es} hosts>>
+* Install an {agent} and enroll it in an agent policy containing the {fleet-server} integration to <<add-fleet-server-on-prem-add-server,Add {fleet-server}>>
+
+NOTE: You can install only a single {agent} per host, which means you cannot run
+{fleet-server} and another {agent} on the same host unless you deploy a
+containerized {fleet-server}.
+
+[discrete]
+[[add-fleet-server-on-prem-compatibility]]
+== Compatibility and prerequisites
+
+{fleet-server} is compatible with the following Elastic products:
+
+* {stack} 7.13 or later.
+** For version compatibility: {es} >= {fleet-server} >= {agent} (except for
+bugfix releases)
+** {kib} should be on the same minor version as {es}.
+
+* {ece} 2.9 or later
++
+--
+** Requires additional wildcard domains and certificates (which normally only
+cover `*.cname`, not `*.*.cname`). This enables us to provide the URL for
+{fleet-server} of `https://.fleet.`.
+** The deployment template must contain an {integrations-server} node.
+--
++
+For more information about hosting {fleet-server} on {ece}, refer to
+{ece-ref}/ece-manage-integrations-server.html[Manage your {integrations-server}].
+
+[discrete]
+[[add-fleet-server-on-prem-hosts]]
+== Add hosts
+
+// What is it / why do you need it?
+Start by adding one or more {fleet-server} hosts.
+A {fleet-server} host is a URLs your {agent}s will use to connect to a {fleet-server}.
+
+// Prereqs
+{fleet-server} hosts should meet the following requirements:
+
+* All agents can connect to the host.
+* The host also has a route to the {es} you plan to use, whether hosted by Elastic
+or deployed on-premises.
+// Need link to "minimum compute resource requirements"
+* The host meets the minimum compute resource requirements based on the maximum number
+of agents you plan to support in your deployment.
+
+// How do you set it up?
+To add a {fleet-server} host:
+
+. In {kib}, go to *Management* -> *{fleet}* -> *Settings*.
+For more information about these settings, see
+{fleet-guide}/fleet-settings.html[{fleet} settings].
+// lint ignore fleet-server
+. Under *Fleet Server hosts*, click *Edit hosts* and specify one or more host
+URLs your {agent}s will use to connect to {fleet-server}. For example,
+`https://192.0.2.1:8220`, where `192.0.2.1` is the host IP where you will
+install {fleet-server}. Save and apply your settings.
++
+TIP: If the **Edit hosts** option is grayed out, {fleet-server} hosts
+are configured outside of {fleet}. For more information, refer to
+{kibana-ref}/fleet-settings-kb.html[{fleet} settings in {kib}].
+
+. In the **{es} hosts** field, specify the {es} URLs where {agent}s will send data.
+For example, `https://192.0.2.0:9200`. Skip this step if you've started the
+{stack} with security enabled (you cannot change this setting because it's
+managed outside of {fleet}).
```

Comment:
It wasn't immediately clear to me what the "**Elasticsearch hosts** field" was in the UI. Is it one or both of these?

<img width="1195" alt="Screen Shot 2022-09-15 at 4 39 24 PM" src="https://user-images.githubusercontent.com/10479155/190513630-2b1b19e1-be5e-43ed-a0a7-d182c4ccc7a3.png">